scripts/xen/gold_linux/run_alpine

```diff
@@ -4,9 +4,12 @@ sudo ln $(dirname $0)/alpine.cfg /etc/xen/alpine.cfg
 
 # Getting alpine linux image
 sudo mkdir /data && cd /data
```

Comment:
How about to use a subdir in $EMBOX/build/... ? That will persist the image for a while